build_tools/azure/install.sh

```diff
@@ -60,15 +60,16 @@ if [[ "$DISTRIB" == "conda" ]]; then
         conda config --set restore_free_channel true
     fi
 
+	make_conda $TO_INSTALL
+
     if [[ "$PYTEST_VERSION" == "*" ]]; then
-        pip install pytest
+        python -m pip install pytest
     else
-        pip install pytest=="$PYTEST_VERSION"
+        python -m pip install pytest=="$PYTEST_VERSION"
     fi
 
-	make_conda $TO_INSTALL
     if [[ "$PYTHON_VERSION" == "*" ]]; then
-        pip install pytest-xdist
+        python -m pip install pytest-xdist
```

Comment:
From `pytest-xdist` [setup.py](https://github.com/pytest-dev/pytest-xdist/blob/master/setup.py), I think it would leave pytest along if `pytest>=4.4.0`.

Reply:
I see what you mean now... if pytest was not installed this would have installed it anyways